src/writer.js

```diff
@@ -246,7 +250,10 @@ define([
                 opts.writeCustomXML(xmlWriter, mug);
             }
             if (opts.writeControlRefAttr) {
-                xmlWriter.writeAttributeString(opts.writeControlRefAttr, mug.absolutePath);
+                var hashtag = mug.hashtagPath;
+                if (hashtag) {
```

Comment:
when would this be false? seems like the `if` is not needed since it was always written before